docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: 'Tutorial 10: Account Updates'
+hide_title: true
+sidebar_label: 'Tutorial 10: Account Updates'
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but
+zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+:::note
+
+This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.9.3.
+
+:::
+
+# Tutorial 10: Account Updates
+
+## Permissions, Preconditions, and Composability
+
+## Overview:
+
+Each zkApp transaction constructed in SnarkyJS is composed of [AccountUpdates](../snarkyjs-reference/classes/AccountUpdate), a set of instructions for the Mina network.
+
+Each AccountUpdate can make assertions about its account, apply updates to its account, as well as make assertions about its child AccountUpdates.
+
+Transactions are structured as a list of trees of AccountUpdates - applied with a ["pre-order"](https://en.wikipedia.org/wiki/Tree_traversal) traversal.
+
+Many of the core features of zkApps - permissions, preconditions, composability, and tokens - are each implemented through the use of AccountUpdates. In this tutorial, you learn all of these except tokens and assertions on account update layouts.
+
+To learn more, see these advanced SnarkyJS docs:
+
+* [Permissions](../advanced-snarkyjs/permissions)
+* [Preconditions](../advanced-snarkyjs/on-chain-values)
+
+## AccountUpdate contents
+
+Each [AccountUpdate](../snarkyjs-reference/classes/AccountUpdate) has these components:
+
+* `PublicKey`: The account address for the account update
+* `TokenId`: By default, the Mina TokenId. Together, TokenId and PublicKey uniquely determine the account.
+* `Preconditions`: What needs to be true for the AccountUpdate to be applied. Corresponds to assertions in a SnarkyJS method.
+* `Updates`: Things that are changed by the AccountUpdate. Can include the zkApp state, permissions, and verification key.
```

Comment:
```suggestion
* `Updates`: Things changed by the AccountUpdate, such as include the zkApp state, permissions, and verification key.
```

Rephrased because it's not a fully inclusive list.